integration-tests/appsec/endpoints-collection.spec.js

```diff
@@ -73,16 +52,83 @@ describe('Endpoints collection', () => {
       { method: 'HEAD', path: '/api/sub/deep' },
       { method: 'POST', path: '/api/sub/deep/:id' },
 
-      // Wildcard routes
-      { method: 'GET', path: '/wildcard/*' },
-      { method: 'HEAD', path: '/wildcard/*' },
-      { method: 'GET', path: '*' },
-      { method: 'HEAD', path: '*' },
-
       { method: 'GET', path: '/later' },
       { method: 'HEAD', path: '/later' },
     ]
 
+    if (framework === 'fastify') {
+      expectedEndpoints.push(
+        // Route with regex - not supported in express5
+        { method: 'DELETE', path: '/regex/:hour(^\\d{2})h:minute(^\\d{2})m' },
+
+        { method: 'OPTIONS', path: '/users/:id?' }, // Added with addHttpMethod
```

Comment:
wrong copy paste again :p
```suggestion
        { method: 'OPTIONS', path: '/users/:id?' },
```